src/guides/optimism-bridge-widget/index.md

```diff
@@ -0,0 +1,1877 @@
+          <div>
+            <div className="txt-lg font-bold">Bridge your ETH!</div>
+          </div>
+        <div className="my-4 space-y-4">
+          <input placeholder="ETH bridged to Optimism" ref={depositRef} type="text" className="focus:outline-none bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-teal-400 focus:border-teal-400 block w-full p-2.5"/>
+          <button onClick={depositEth} className="px-4 py-1 text-sm text-teal-400 font-semibold rounded-lg border border-teal-400 hover:text-white hover:bg-teal-400 hover:border-transparent">Deposit</button>
+          {l1Balance.loading && <div>Loading Goerli Eth Balance...</div>}
+          {l1Balance.error && <div className="font-semibold">Error: {l1Balance.error.message}</div>}
+          {
+            l1Balance.result && (
+                <div>
+                  <span className="text-slate-500">Current Goerli Eth Balance: </span>
+                  <span className="font-semibold">{l1Balance.result}</span>
+                </div>
+            )
+          }
+          <input placeholder="ETH bridged to Ethereum"ref={withdrawRef} type="text" className="focus:outline-none bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-teal-400 focus:border-teal-400 block w-full p-2.5"/>
+          <button onClick={withdrawEth} className="px-4 py-1 text-sm text-teal-400 font-semibold rounded-lg border border-teal-400 hover:text-white hover:bg-teal-400 hover:border-transparent">Withdraw</button>
+          {l2Balance.loading && <div>Loading Optimism Goerli Eth Balance...</div>}
+          {l2Balance.error && <div className="font-semibold">Error: {l2Balance.error.message}</div>}
+          {
+              l2Balance.result && (
+                <div>
+                  <span className="text-slate-500">Current Optimism Goerli Eth Balance: </span>
+                  <span className="font-semibold">{l2Balance.result}</span>
+                </div>
+              )
+            }
+          </div>
+        </div>
+    </div>
+  );
+}
+
+export default Bridge;
+```
+
+</p>
+</details>
+
+The things to note here is that we effectively copy pasted the relevant depositETH and withdrawETH content from `scripts/goerli_bridge_value.js`. There are a few key differences though, namely:
+
+- We split `reportBalances` into `getL1Eth` and `getL2Eth` to be displayed on our web page.
+- We needed to convert the input values for withdraw and deposit from `text` to `BigNumber` (`ethers.utils.parseEther(withdrawRef.current.value)` and `ethers.utils.parseEther(depositRef.current.value)`)
+
+Finally, the last piece is to make sure we add this page to our list of pages in `_app.js`! To the end of the links, add:
+
+```javascript
+<Link href="/goerli-bridge">
+  <a className="mr-6 text-teal-400">
+    Goerli Bridge
+  </a>
+</Link>
+```
+
+Now, everything should be hooked up! Run `npm run dev` from the client folder and try it out!
```

Comment:
Do we have the full working code on a repo / branch somewhere that we could include if folks want to fast track things?

Reply:
Also possibly a hosted version of it could be pretty cool too (we can discuss options for where we might put it).

Reply:
yep! the link to the code was at the start: https://github.com/trufflesuite/unleashed_optimism_bridge